app/controllers/issues_controller.rb

```diff
@@ -111,11 +111,15 @@ def destroy
 
   def import
     importer = IssueImporter.new(params)
-    @results = importer.query()
+    results = importer.query()
```

Comment:
https://github.com/rubocop/ruby-style-guide#method-call-with-no-arguments

```suggestion
    results = importer.query
```